GWAS/PCA.ipynb

```diff
@@ -735,6 +737,16 @@
     "        colnames(pheno) = c(\"FID\", \"IID\", \"MID\", \"PID\", \"SEX\", \"STATUS\")\n",
     "    } else {\n",
     "        pheno <- read.table(${_input[1]:r}, header=T,stringsAsFactors =F)\n",
+    "        if('IID' %in%  colnames(pheno) == FALSE){\n",
```

Comment:
what if I have a file with header

```
IID pheno
... ...
```

Is it still going to work? what if my phenotype file is:

```
sample_id pheno
```

?